lib/MySQL_Thread.cpp

```diff
@@ -2961,13 +2961,23 @@ void MySQL_Thread::run___get_multiple_idle_connections(int& num_idles) {
 
 // this function was inline in MySQL_Thread::run()
 void MySQL_Thread::ProcessAllMyDS_BeforePoll() {
+	bool check_if_move_to_idle_thread = false;
+#ifdef IDLE_THREADS
+	if (GloVars.global.idle_threads) {
+		if (curtime > last_move_to_idle_thread_time + (unsigned long long)mysql_thread___session_idle_ms * 1000) {
+				last_move_to_idle_thread_time=curtime;
+			check_if_move_to_idle_thread=true;
+		}
+	}
+#endif
 	for (unsigned int n = 0; n < mypolls.len; n++) {
 		MySQL_Data_Stream *myds=NULL;
 		myds=mypolls.myds[n];
 		mypolls.fds[n].revents=0;
 		if (myds) {
 #ifdef IDLE_THREADS
-			if (GloVars.global.idle_threads) {
+			//if (GloVars.global.idle_threads) {
```

Comment:
Remove?

Reply:
:) yes, I have been lazy. Will remove it!
Thank you for pointing it